projects/burnr/src/pages/Home.tsx

```diff
@@ -1,40 +1,55 @@
 import React from 'react';
-import styled from 'styled-components';
 
-import { CardHeader, Card, CardContent, CardMedia, Grid } from '@material-ui/core';
+import { Grid, Paper, Divider, IconButton, Box } from '@material-ui/core';
+import VisibilityIcon from '@material-ui/icons/Visibility';
 
-import { useChainInfo } from '../hooks';
+import { NavTabs, LogoApp, NodeSelector, AccountCard, BalanceValue, NavFooter } from '../components';
 
-interface Props {
-  className?: string;
-}
+function Home ():  React.ReactElement {
 
-function Home ({ className }: Props):  React.ReactElement<Props> {
-	const blockHash = useChainInfo();
+	return (
+		<>
+			<Grid container alignItems='center'>
```

Comment:
I'd put this part (`<LogoApp>` & `<NodeSelector>`) in the `<ThemeHeader>`, because they're both relevant or all pages (even though we just have a single one for now).

It possibly makes sense, to import all 4 components in the `<ThemeHeader>` component

So in ThemeHeader.tsx:
```
<div className={classes.root}>	
	<LogoSubstrate theme={theme} />
        <LogoApp>
        <NodeSelector>
	<ThemeButton theme={theme} onClick={() => setTheme(!theme)} />
</div>
```

Also: The header also  looks a little broken on smaller screens right now, the elements are overlapping.
<img width="500" alt="Screenshot 2020-08-03 at 11 00 08" src="https://user-images.githubusercontent.com/125398/89166400-47781580-d57a-11ea-8bf7-dd8e49b3c525.png">